polls/templates/index.html

```diff
@@ -0,0 +1,109 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        {% load static %}
+        <meta charset="utf-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <title>Skalierbare Systeme Hausaufgabe</title>
+        <meta name="description" content="">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
```

Comment:
Favicon should be in the static folder
```suggestion
        <meta name="viewport" content="width=device-width, initial-scale=1">
        <link rel="shortcut icon" href="{%  static 'favicon.ico' %}">
```